Makefile

```diff
@@ -8,10 +8,10 @@ GOBIN = ./build/bin
 GO ?= latest
 GORUN = env GO111MODULE=on go run
 
-geth:
+suave:
 	$(GORUN) build/ci.go install ./cmd/geth
 	@echo "Done building."
-	@echo "Run \"$(GOBIN)/geth\" to launch geth."
+	@echo "Run \"$(GOBIN)/geth\" to launch suave."
```

Comment:
shouldn't this be Run \"$(GOBIN)/suave\ to launch suave"? Which additionally requires modifying to make sure the build outputs to that directory. Also we typically stylize SUAVE but not sure thats necessary here just a thought